content/applications/hr/employees/new_employee.rst

```diff
@@ -297,54 +322,54 @@ however, some information in this section may be critical for the company's payr
   receiving any type of commuter benefits. Lastly, enter the employee's license plate information in
   the :guilabel:`Private Car Plate` field.
-- :guilabel:`Education`: Select the highest level of education completed by the employee from the
-  :guilabel:`Certificate Level` drop-down menu. Options include :guilabel:`Graduate`,
-  :guilabel:`Bachelor`, :guilabel:`Master`, :guilabel:`Doctor`, or :guilabel:`Other`. Type in the
-  :guilabel:`Field of Study`, and the name of the :guilabel:`School` in the respective fields.
-- :guilabel:`Work Permit`: If the employee has a work permit, enter the information in this section.
-  Type in the :guilabel:`Visa No` (visa number) and/or :guilabel:`Work Permit No` (work permit
-  number) in the corresponding fields. Using the calendar selector, select the :guilabel:`Visa
-  Expire Date` and/or the :guilabel:`Work Permit Expiration Date` to enter the expiration date(s).
-  If available, upload a digital copy of the work permit document. Click :guilabel:`Upload Your
-  File`, navigate to the work permit file in the file explorer, and click :guilabel:`Open`.
-- :guilabel:`Family Status`: Select the current :guilabel:`Marital Status` using the drop-down menu,
+- :guilabel:`Family Status`: select the current :guilabel:`Marital Status` using the drop-down menu,
   either :guilabel:`Single`, :guilabel:`Married`, :guilabel:`Legal Cohabitant`, :guilabel:`Widower`,
   or :guilabel:`Divorced`. If the employee has any dependent children, enter the :guilabel:`Number
   of Dependent Children` in the field.
-- :guilabel:`Emergency`: Type in the :guilabel:`Contact Name` and :guilabel:`Contact Phone` number
+- :guilabel:`Emergency`: type in the :guilabel:`Contact Name` and :guilabel:`Contact Phone` number
   of the employee's emergency contact in the respective fields.
-- :guilabel:`Citizenship`: This section contains all the information relevant to the citizenship of
+- :guilabel:`Education`: select the highest level of education completed by the employee from the
+  :guilabel:`Certificate Level` drop-down menu. Default options include :guilabel:`Graduate`,
+  :guilabel:`Bachelor`, :guilabel:`Master`, :guilabel:`Doctor`, or :guilabel:`Other`. Type in the
+  :guilabel:`Field of Study`, and the name of the :guilabel:`School` in the respective fields.
+- :guilabel:`Work Permit`: if the employee has a work permit, enter the information in this section.
+  Type in the :guilabel:`Visa No` (visa number) and/or :guilabel:`Work Permit No` (work permit
+  number) in the corresponding fields. Using the calendar selector, select the :guilabel:`Visa
+  Expiration Date` and/or the :guilabel:`Work Permit Expiration Date` to enter the expiration
+  date(s). If available, upload a digital copy of the :guilabel:`Work Permit` document. Click
+  :guilabel:`Upload your file`, navigate to the work permit file in the file explorer, and click
+  :guilabel:`Open`.
```

Comment:
```suggestion
  :guilabel:`Bachelor`, :guilabel:`Master`, :guilabel:`Doctor`, :guilabel:`Other`, or
  :guilabel:`Master:Civil Engineering`.
  
  Type in the :guilabel:`Field of Study`, and the name of the :guilabel:`School` in the respective
  fields.
- :guilabel:`Work Permit`: if the employee has a work permit, enter the information in this section.
  Type in the :guilabel:`Visa No` (visa number), and/or :guilabel:`Work Permit No` (work permit
  number), in the corresponding fields.
  
  Using the calendar selector, select the :guilabel:`Visa Expiration Date`, and/or the
  :guilabel:`Work Permit Expiration Date`, to enter the expiration date(s).
  
  If available, upload a digital copy of the :guilabel:`Work Permit` document. Click
  :guilabel:`Upload your file`, navigate to the work permit file in the file explorer, and click
  :guilabel:`Open`.
```

Reply:
The Master:Civil Engineering only appears in Runbots, and it's data they added/imported for whatever testing reasons. If you make a blank database with all the HR apps (and the relevant ones needed, documents, accounting, planning, etc) that does not appear, and what is in the original list are all that show. That's the reason I did not add it.